### PR TITLE
docs: add Go learning resource in Bahasa Indonesia

### PR DESCRIPTION
**Pull request by danielcristho**
## What does this PR do?
Add resource(s) | Remove resource(s) | Add info | Improve repo

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: books/free-programming-books-id.md [hunk 1]
```diff
@@ -55,6 +55,7 @@
 ### Git
 
 * [Belajar Git untuk Pemula](https://github.com/petanikode/belajar-git) - Ahmad Muhardian, Petani Kode (HTML)
+* [Dasar Pemrograma Golang](https://santrikoding.com/ebook/dasar-pemrograman-golang) - Ebook dari Santri Koding
```
Comment (eshellman) [inline books/free-programming-books-id.md:58]:
```suggestion
* [Dasar Pemrograma Golang](https://santrikoding.com/ebook/dasar-pemrograman-golang) - Noval Agung Prayogo
```

Comment (eshellman) [reply books/free-programming-books-id.md]: am i wrong?

Comment (danielcristho) [reply books/free-programming-books-id.md]: yep, you're right. Sorry, I forgot to put it in the GO section and there was a typo. But I've fixed it.

Thanks.

**Comment by eshellman** [comment]
add the author

**Pull request closed without merging**